Classes/Command/ReferenceIndexCommandController.php

```diff
@@ -24,13 +24,44 @@ class ReferenceIndexCommandController extends CommandController
     /**
      * Update Reference Index
      *
+     * Updates the reference index - if providing the -f parameter the
+     * indexing will index directly to sys_refindex - else the
+     *
+     * @return void
+     */
+    public function updateCommand() {
+        if (in_array('-f', $_SERVER['argv'])) {
```

Comment:
These should be passed as arguments for the `updateCommand` method, the `-f` etc. short forms are against common practice. I suggest `$force`, `$check` and `$silent` all booleans, meaning you specify `--force 1 --check 1` on the command line. This also means you don't need the default "options" output; the usual CLI "help" command will inspect the arguments and list them for you.